### DISCUSSION TITLE
Unable to sync pages directory

### DISCUSSION THREAD

**Discussion by scao53** [community/community#45797] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

Two of my websites have had issues "Unable to sync pages directory." when I tried to change things on the websites since yesterday evening. They didn't have issues before. Could you please help with these?

Thanks so much for your help and time!

Best,
Shiya

**Comment by jge162** [comment]
Can you offer more context, or details. Unable to help with current details. Thx.

**Reply by scao53** [reply]
For example, the master branch is where the page is built.

When I changed something in index.html, the change didn't show up in my website. And I received an email from GitHub which said "Unable to sync pages directory."

**Reply by jge162** [reply]
Ensure you have committed and pushed your changes to the correct branch on GitHub.

**Reply by lltx** [reply]
@jge162  I also encountered the same problem, the content has not changed, but it still cannot be built, suggesting 

```
The page build failed for the `master` branch with the following error:

Unable to sync pages directory. Please try again later.

```

repo: https://github.com/datavaa/datavaa.github.io

**Reply by jge162** [reply]
@scao53 @lltx You can submit a ticket here for support and they will help you too [Github Support](https://support.github.com/request/account?tags=hubberfy_account)

**Reply by jge162** [reply]
Choose the option that bests suits your scenario.

**Reply by jge162** [reply]
- Other support

**Reply by lltx** [reply]
Thank you

**Reply by mduong26** [reply]
This is also happening to me. It has been working all fine for the past three months. Today I made a small change to a markdown file and now I have this problem too.

**Reply by luisaras** [reply]
Same here. Website was working just fine, then I made a small change in the text and now I receive this same error.

**Reply by danielleroux** [reply]
Same here last week published the documentation. Today want to fix a typo but pages is not synced anymore. Received email:

```
The page build failed for the `gh-pages` branch with the following error:

Unable to sync pages directory. Please try again later.

For information on troubleshooting Jekyll see:

  https://docs.github.com/articles/troubleshooting-jekyll-builds
```

What i have already tested:

- Make change to index.html
- Create new branch and switch branch in settings

Nothing works.

**Reply by scao53** [reply]
It works for me now. I submitted a ticket as @jge162 suggested. The issue seems to have possibly been related to [this incident](https://www.githubstatus.com/incidents/zf2tk6x40gmh).

**Reply by jge162** [reply]
Glad to hear your back up and running. Have a great rest of your week.

**Comment by MostafaWaleed0** [comment]
If you are facing an issue with syncing changes made to your website's Pages directory, here are some steps you can try to resolve the issue:

<ol><li>Check your internet connection.</li><li>Clear your browser cache and cookies.</li><li>Restart your browser and try making the changes again.</li><li>Check if there are any updates available for your browser.</li><li>Try using a different browser to make the changes.</li><li>Check if there are any recent changes to the website's code that might be causing the issue.</li><li>Contact the website support or the person responsible for maintaining the website for assistance.</li></ol>
